.github/workflows/build.yml

```diff
@@ -24,24 +24,24 @@ jobs:
           node-version: '12'
 
       - name: Setup Hugo
-        uses: peaceiris/actions-hugo@v2
-        with:
-          hugo-version: '0.76.5'
+        run: ../bin/installHugo.sh
```

Comment:
Why do we pass by a script rather a github actions on this repository ?

I don't understand. We use on all our CI multiple setup actions (node, hugo, apache, mysql, node, php, python, google).

Reply:
Because you can't compare node, or php with hugo.
Hugo is a software, exactly like php-cs-fixer. And we are not using php-cs-fixer github actions because we want to rely on the same version as we define in the composer.json to avoid duplicate.

Reply:
But Hugo is comparable to Apache or MySQL.

Reply:
Nope, it's comparable to Symfony :)

Reply:
or if you prefer, Django or RoR :)

Reply:
Sorry @PierreRambaud. For one time, I disapprove. I prefer use of Github Actions (more simple, more clear).

Reply:
I'm not against any of the 2 solutions but if we chose @PierreRambaud's one, we should update the docker-compose as well because I think right now it's using the `master` branch of hugo.

Reply:
Three versions to maintain, not sure it's the best idea :p